.ci/create-github-release.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: $BASH_SOURCE <version>"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+checkForVariable "GITHUB_TOKEN"
+
+curl -i -H "Authorization: token $GITHUB_TOKEN" \
+  -d "{ \"tag_name\": \"checkstyle-$TARGET_VERSION\", \
```

Comment:
Please use heredoc to make json body more readable/maintainable. See an example in the project at https://github.com/checkstyle/checkstyle/blob/a359c4121aa15aad1cd147e0ebfc1528bfd82bae/.ci/update-github-page.sh#L80, note that you can also just cat the heredoc in a subshell and assign it to some variable, like `X=$(cat EOF {"myjson":{}} EOF)`.

Reply:
Done.